travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -2,16 +2,21 @@
 #  pylint configuration file will display them without affecting your build status.
 
 # Beta message and code:
+#   api-one-deprecated - W8104
```

Comment:
If you use `api.one` in v6 or v7 you will have server errors and pylint fails too.
If you use `api.one` in v8 or v9 you will have just pylint fails.